src/core/rss.rs

```diff
@@ -13,19 +13,37 @@ const ATOM_NAMESPACE: &str = "http://www.w3.org/2005/Atom";
 impl From<SsufidPost> for rss::Item {
     fn from(post: SsufidPost) -> Self {
         let mut builder = ItemBuilder::default();
+
+        let description = {
+            let truncate_idx = post
+                .content
+                .char_indices()
+                .nth(50)
+                .map_or(post.content.len(), |(i, _)| i);
+            if truncate_idx < post.content.len() {
+                let mut truncated = post.content.clone();
+                truncated.truncate(truncate_idx);
+                truncated.push_str("...");
+                truncated
```

Comment:
`map_or_else()` 메서드를 사용해서 짧게 줄일 수 있을 것 같아요!

```suggestion
     let description = post.content.char_indices().nth(50).map_or_else(
            || post.content.clone(),
            |(i, _)| format!("{}...", &post.content[..i]),
        );
```